a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -286,34 +286,48 @@ <h5>Variables setup</h5>
                         </li>
                     	
                     	<li>
-                    		<b>LET</b> <var>has_nonvisual_metadata</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property=("<i>schema:accessMode</i>", "<i>schema:accessModeSufficient</i>")]</code>.
+                    		<b>LET</b> <var>audio_only_content</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessMode</i>" and normalize-space() = "auditory" and count(//meta[@property="schema:<i>accessMode</i>"]) = 1]</code>.
                     	</li>
                     	
-                        <li>
+                    	<li>
+                    		<b>LET</b> <var>braille_only_content</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessMode</i>" and normalize-space() = "tactile" and count(//meta[@property="schema:<i>accessMode</i>"]) = 1]</code>.
+                    	</li>
+                    	
+                    	<li>
                             <b>LET</b> <var>non_textual_content</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="<i>schema:accessMode</i>" and not(normalize-space() = "textual")]</code>.
                         </li>
 
                         <li>
                             <b>LET</b> <var>textual_alternatives</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="<i>schema:accessibilityFeature</i>" and normalize-space()=("<i>longDescription</i>", "<i>alternativeText</i>", "<i>describedMath"</i>, "<i>transcript</i>")]</code>.
                         </li>
+                    	
+                    	<li>
+                    		<b>LET</b> <var>visual_only_content</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessMode</i>" and normalize-space() = "visual" and count(//meta[@property="schema:<i>accessMode</i>"]) = 1]</code>.
```

Comment:
I would personally like to see accessModeSufficient="visual" by itself as that is what we have been telling everyone if this is a picture book that is the main metadata to include.

Reply:
But that won't tell you it's a visual book with no text alternatives. It just tells you that one path through the content is visual.

If a visual work has alt text or text descriptions, it also has a sufficient access mode of "visual, textual". That means there is some text, although it's not fully equivalent as you still need to see the pictures. This test should probably replace trying to pick out accessibility features. In other words, if textual is only one of a combined sufficient access mode sets, then the publication is not fully readable as text.

Reply:
I've modified the algorithm to test both. So if there is non-textual content but there is either a textual combined sufficient access mode or the text alternatives, we say not all the content is readable.

Since full text equivalency is captured earlier on, this should capture the rest of the cases where some text is available. It's a bit of an overlapping test, but ensures that if the publisher got at least their sufficient access modes right or listed all their features the publication won't fall through to no information available.